components/com_contact/src/Controller/ContactController.php

```diff
@@ -179,14 +179,26 @@ public function submit()
 		// Validation succeeded, continue with custom handlers
 		$results = $this->app->triggerEvent('onValidateContact', array(&$contact, &$data));
 
+		$passValidation = true;
+
 		foreach ($results as $result)
 		{
 			if ($result instanceof \Exception)
 			{
-				return false;
+				$passValidation = false;
+				$app->enqueueMessage($result->getMessage(), 'error');
 			}
 		}
 
+		if(!$passValidation)
```

Comment:
```suggestion
		if (!$passValidation)
```
Fix code style error.